### PR TITLE
Added the EBF and CF Forms

### PR DESCRIPTION
**Pull request by ShubhamBansal1997**
- Added the EBF Form
- Added the CF Form
- Minor fixes

### CODE REVIEW

File: data_gen/data/templates/forms/cf-form.json [hunk 1]
```diff
@@ -0,0 +1,1015 @@
...
+        "@uiVersion": "1",
+        "@version": "31733",
+        "@xmlns": "http://openrosa.org/formdesigner/792DAF2B-E117-424A-A673-34E1513ABD88",
+        "add": "2019-02-18",
+        "app_scope": "cas",
+        "case_load_ccs_record0": {
+            "case": {
+                "@case_id": "%(pregnant_ccs_record_case_id)s",
+                "@date_modified": "2019-11-19T10:32:04.642000Z",
+                "@user_id": "ed16783aa2f041078d98ea83b15e2525",
```
Comment (czue) [inline data_gen/data/templates/forms/cf-form.json]: nit: `%(user_id)s`

File: data_gen/data/templates/forms/ebf-form.json [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+        "safe": "yes",
+        "show_mother_death_questions": "no",
+        "show_mother_ebf_questions": "yes",
+        "success": "OK",
+        "total_children_died": "0",
+        "unscheduled_visit": "0",
+        "update_household_case": {
+            "update_household_case": {
+                "case": {
+                    "@case_id": "a244fcd3-63bb-43eb-b3c6-68976dfadfed",
```
Comment (czue) [inline data_gen/data/templates/forms/ebf-form.json]: nit: `%(household_case_id)s`

File: data_gen/data/templates/forms/ebf-form.json [hunk 3]
```diff
@@ -0,0 +1,226 @@
...
+                    "@date_modified": "2020-01-20T10:47:14.467000Z",
+                    "@user_id": "64b358940d02479f98264c27fe125ee7",
```
Comment (czue) [inline data_gen/data/templates/forms/ebf-form.json]: also `%(user_id)s` (though I don't think this matters in practice)

File: data_gen/data/templates/forms/cf-form.json [hunk 4]
```diff
@@ -0,0 +1,1015 @@
...
+                    "item": [
```
Comment (czue) [inline data_gen/data/templates/forms/cf-form.json]: is all this stuff used?

Comment (ShubhamBansal1997) [reply data_gen/data/templates/forms/cf-form.json]: Not sure, I just copied from the sample case, might be used in future to refer something

**Comment by ShubhamBansal1997** [comment]
@czue fixed the comments, let's merge this!!

**Comment by czue** [comment]
you're welcome to merge once approved ;)

**Pull request merged**